ndbench-core/src/main/java/com/netflix/ndbench/core/generators/SlidingWindowFlipStringKeyGenerator.java

```diff
@@ -1,5 +1,6 @@
 /*
  *  Copyright 2016 Netflix, Inc.
+ *  Portions copyright 2018 Amazon Web Services, Inc.
```

Comment:
I am fine with the copyright here.  I was wondering as the project scales the number of licenses will increase on a per-file basis. For example, assume 10 developers working on this project, then the copyrights will have to be populated on a per file basis. I was thinking whether we can follow an approach like the following

https://github.com/caolan/nodeunit/blob/master/CONTRIBUTORS.md

and in the Copyright have something like

```
Copyright YYYY Netflix, Inc and NDBench contributors  
```
with a direct link to the [CONTRIBUTORS.md](https://github.com/Netflix/ndbench/blob/master/CONTRIBUTORS.md)